cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -1284,4 +1291,15 @@ protected void addContainerEnvsToExistingEnvs(List<EnvVar> existingEnvs, List<Co
             }
         }
     }
+
+    protected String getJavaSystemPropertiesToString(List<SystemProperty> javaSystemProperties) {
```

Comment:
because you are passing the `javaSystemProperties` this method could be static but I think we are using the `ModelUtils` class for this kind of things so maybe we should move the method there.